scheduler.c

```diff
@@ -108,6 +110,12 @@ rb_fiber_scheduler_make_timeout(struct timeval *timeout)
     return Qnil;
 }
 
+VALUE
+rb_fiber_scheduler_timeout_raise(VALUE scheduler, VALUE timeout)
+{
+    return rb_check_funcall(scheduler, id_timeout_raise, 1, timeout);
```

Comment:
This doesn't pass the block and doesn't match the arguments of `timeout_raise` above.
Probably best to remove until it's tested.
Also, I see no potential usage for it as a C API.

Reply:
Looks good to me except this.

Reply:
Thanks, I didn't realise it had diverged so much, I'll fix it.